iterate.md

```diff
@@ -0,0 +1,688 @@
+---
+title: iterate
+---
+
+`iterate` is a powerful iteration facility for Common Lisp, and a alternative to `loop`. As opposed to `loop`, `iterate` is more lispy, extensible, besides being more powerful.
+
+See [The Iterate Manual] for a more detailed explanation of the various facilities. Also check [Other Resources on Iterate](#other-resources-on-iterate).
+
+## Highlights
```

Comment:
I find a bit rough to see high-level highlights straight away, without examples beforehand. Maybe, do you mean a reader comes here after having read the examples of the "iteration" page ?